src/backend/defs/rest-api/paths/messages.yaml

```diff
@@ -0,0 +1,248 @@
+---
+messages:
+  get:
+    description: Returns the list of messages for current user according to given
+      parameters/filter
+    tags:
+    - messages
+    security:
+    - basicAuth: []
+    parameters:
+    - name: X-Caliopen-PI
+      in: header
+      required: true
+      description: The PI range requested in form of `0;100`
+      type: string
+      default: 0;100
+    - name: discussion_id
+      in: query
+      description: filter messages belonging to a specific discussion
+      required: true
+      type: string
+    - name: limit
+      in: query
+      required: false
+      type: integer
+      description: number of messages to return per page
+    - name: offset
+      in: query
+      type: integer
+      required: false
+      description: number of pages to skip from the response
```

Comment:
yes, we should definitively add more filtering/ordering options on this route, but for now, even the « offset » option does not already work… WIP.